contracts/contracts/MACI.sol

```diff
@@ -15,40 +15,39 @@ import {SnarkConstants} from "./crypto/SnarkConstants.sol";
 
 import {Ownable} from "@openzeppelin/contracts/access/Ownable.sol";
 
-/*
+/**
  * Minimum Anti-Collusion Infrastructure
  * Version 1
  */
 contract MACI is IMACI, DomainObjs, Params, SnarkCommon, Ownable {
-    // The state tree depth is fixed. As such it should be as large as feasible
-    // so that there can be as many users as possible.  i.e. 5 ** 10 = 9765625
-    // this should also match the parameter of the circom circuits.
+    /// @notice The state tree depth is fixed. As such it should be as large as feasible
```

Comment:
Why the triple slash? I'm guessing this was not intended.

Reply:
https://docs.soliditylang.org/en/v0.8.16/natspec-format.html - you can define single line natspec comments with /// and multiline with /// or / ** */

Reply:
Ah, ok, thanks for the explanation 👌